Applications/Spire/Include/Spire/Charting/ColorPicker.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef SPIRE_COLOR_PICKER_HPP
+#define SPIRE_COLOR_PICKER_HPP
+#include <QLabel>
+#include <QWidget>
+#include "Spire/Charting/Charting.hpp"
+
+namespace Spire {
+
+  //! Displays a color picker and signals when a color is selected.
+  class ColorPicker : public QWidget {
+    public:
+
+      //! Signals that a color has been interacted with.
+      using ColorSignal = Signal<void (const QColor& color)>;
```

Comment:
Document the parameters to the signal. What does it mean to "interact" with a color?